### DISCUSSION TITLE
Smart Filter lists (similar to Plex or ComicRack)

### DISCUSSION THREAD

**Discussion by Cantello** [gotson/komga#1438] [Ideas]
It would be great to have a more granular seapration than libraries without the need to go into each library and filter there or have a collection (to which you need to manually add comics).
Expand the already present filter functiuon by a few more metadata fields (count, published status & format come to my mind) and have them shown as collapsible on the sidebar below libraries.

**Comment by gotson** [comment]
Duplicate of #828
